packages/bpk-component-calendar/src/utils.js

```diff
@@ -21,21 +21,33 @@ import {
   calendarDaySpacing,
 } from 'bpk-tokens/tokens/base.es6';
 
-const remToPx = value => {
-  let parsed = null;
+const splitToken = value => {
+  const match = value.match(/(^[+-]?(?:\d*\.)?\d+)(.+)/i);
+  if (!match) {
+    throw new Error(
+      `Invalid token value. Expecting a valid css unit, got ${value}`,
+    );
+  }
+  const [_, val, unit] = match; // eslint-disable-line no-unused-vars
+  return [parseFloat(val), unit];
+};
 
-  if (/rem$/.test(value)) {
-    parsed = parseFloat(value.replace(/rem/, '')) * 16;
+export const getCalendarGridWidth = (multiplier = 1) => {
+  const [sizeValue, sizeUnit] = splitToken(calendarDaySize);
+  const [spacingValue, spacingUnit] = splitToken(calendarDaySpacing);
+
+  if (sizeUnit !== spacingUnit) {
+    throw new Error(
+      `'calendarDaySize' and 'calendarDaySpacing' must use the same unit. Got ${sizeUnit} and ${spacingUnit}`,
+    );
   }
 
-  return parsed || null;
+  const width = multiplier * (7 * (sizeValue + spacingValue));
```

Comment:
Using `calc` would be ideal but it's not working on IE11 (even though it says it does). I reckon the problem is `translateX` plus `calc`.